soco/core.py

```diff
@@ -1210,8 +1210,10 @@ def parse_zone_group_member(member_element):
         # and the set of all members
         self._all_zones.clear()
         self._visible_zones.clear()
+        # Compatibilty fix for pre-10.1 firmwares
```

Comment:
Typo: Compatibilty -> Compatibility

Reply:
You should probably mention which branch is the compatibility one.